fbuapp/View Controllers/FilterViewController.m

```diff
@@ -0,0 +1,43 @@
+//
+//  FilterViewController.m
+//  fbuapp
+//
+//  Created by danielavila on 7/31/19.
+//  Copyright © 2019 eazheng. All rights reserved.
+//
+
+#import "FilterViewController.h"
+#import "Post.h"
+
+
+
+@interface FilterViewController ()
+
+@end
+
+@implementation FilterViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    UIBarButtonItem *myButton = [[UIBarButtonItem alloc]init];
+    
+    myButton.action = @selector(presentHomeViewController:);
+    myButton.title = @"Cancel";
+    myButton.target = self;
+    self.navigationItem.leftBarButtonItem = myButton;
+}
```

Comment:
Here's another way to create a navigation bar button:
self.navigationItem.leftBarButtonItem = [[UIBarButtonItem alloc] initWithTitle:@"   " style:UIBarButtonItemStylePlain target:self action:@selector(     )];